src/styles/components/_component-card.scss

```diff
@@ -1,79 +1,84 @@
-.component-link {
-  display: block;
-  outline: none;
+.component-card {
+  display: flex;
+  flex-direction: column-reverse;
+  height: 21.875rem; // 350px/16
   width: 100%;
-  height: 100%;
+  cursor: pointer;
+  transition: all 150ms ease-in-out;
+  border: 2px solid var(--foreground-color);
 
-  .component-card {
-    display: flex;
-    flex-direction: column-reverse;
-    height: 21.875rem; // 350px/16
-    width: 100%;
-    cursor: pointer;
+  .component-link:focus & {
+    outline: none;
+    border: 2px solid var(--accent-color);
+  }
+
+  &__image {
+    box-sizing: border-box;
     transition: all 150ms ease-in-out;
-    border: 2px solid var(--foreground-color);
+    width: 100%;
+    height: 70%;
+    padding: min(15%, 2rem);
+    margin: auto;
+  }
 
-    &__image {
-      box-sizing: border-box;
-      transition: all 150ms ease-in-out;
-      width: 100%;
-      height: 70%;
-      padding: min(15%, 2rem);
-      margin: auto;
-    }
+  @media (prefers-color-scheme: dark) {
+    border: 2px solid var(--background-color);
+  }
```

Comment:
From line 15-26: 

Because you're using a new image class (`&__image-open`), I think all of these properties need to go there instead. I deleted them from `&__image` and put them on after the `display: block` you have for `&__image-open`.